### ISSUE TITLE
Conflict with AuthorizeView tag and ApexCharts?

### ISSUE THREAD

**Issue by ldstenor** [apexcharts/Blazor-ApexCharts#177]
I'm having a problem once I add the @using ApexCharts to my MudBlazor project.  

The error I get is "Cannot convert lambda expression to intended delegate type because some of the return types in the block are not implicitly convertible to the delegate return type" and it references a file: "Pages_Admin_PackagesStats_razor.g.cs" at line 237.

The line at 237 in this file:
__builder.AddAttribute(13, "Authorized", (Microsoft.AspNetCore.Components.RenderFragment<Microsoft.AspNetCore.Components.Authorization.AuthenticationState>)((Auth) => (__builder2) => {
                __builder2.OpenComponent<Tracker2.Pages.Admin.AdminNavigation>(14);

Is ApexCharts somehow clashing with my AuthorizeView tag on that page?  Any help is appreciated!

**Comment by joadan** [comment]
Hello,

This sounds strange, could you post you code where you get the error.

**Comment by ldstenor** [comment]
I think I got it figured out.  The error message I received wasn't helpful at all, made me believe it was a conflict with Authorization.  It was not.  I had to start removing components until I found the issue.

Basically, MudBlazor has classes for Color and Align...and so does ApexCharts.  The two were clashing, so the solution was to change this:

<MudText Typo="Typo.h2" Align="Align.Center" Color="Color.Info">My Test Page</MudText>

to this:

<MudText Typo="Typo.h2" Align="MudBlazor.Align.Center" Color="MudBlazor.Color.Info">My Test Page</MudText>

I had to be less ambiguous.

Thanks!
